### ISSUE TITLE
Goals Before Spring Break

### ISSUE THREAD

**Issue by nickkrisulevicz** [Salisbury-University/robot-weed-killer#28]
After consulting with client (Dr. Franchi), our team had a milestone laid out to be reached by spring break:

Priority one: The app needs to be functional and able to toggle between manual and autonomous operation. The app must be able to control the robot in its current state before major hardware upgrades are made. Initial autonomous operation must be achieved in an environment with no obstacles and square boundary. Additional pathfinding and navigation will be implemented later. For now, we need to focus on getting the robot to drive autonomously in a basic setting first.

Next milestone will follow after spring break once current one is done.

**Comment by nickkrisulevicz** [comment]
UPDATE: This milestone was created in milestones, and it should not have been made in issues, this issue can be closed out

**Comment by ZM009** [comment]
No longer needed
